kubernetes/adapter.go

```diff
@@ -395,7 +395,7 @@ func (a *Adapter) UpdateIngressLoadBalancer(ingress *Ingress, loadBalancerDNSNam
 	case TypeIngress:
 		return a.ingressClient.updateIngressLoadBalancer(a.kubeClient, ingress.Namespace, ingress.Name, loadBalancerDNSName)
 	}
-	return fmt.Errorf("Unknown resourceType '%s', failed to update Kubernetes resource", ingress.ResourceType)
+	return fmt.Errorf("unknown resourceType '%s', failed to update kubernetes resource", ingress.ResourceType)
```

Comment:
nit: `Kubernetes` should still be capitalized.

Reply:
I agree 😁 let me fix that.